components/nimbus/src/enrollment.rs

```diff
@@ -372,7 +358,7 @@ impl ExperimentEnrollment {
     /// partipation would submit partial data that could skew analysis.
     ///
     #[cfg_attr(not(feature = "stateful"), allow(unused))]
-    fn reset_telemetry_identifiers(
+    pub fn reset_telemetry_identifiers(
```

Comment:
What's the difference  between this `reset_telemetry_identifiers(..)` and the one in the new `components/nimbus/src/stateful/enrollment.rs` (line 177) that was removed from this file below on line 1384?

Reply:
So the one in `src/enrollment` is a method on the `ExperimentEnrollment` struct. The one in `src/stateful/enrollment` is a standalone function that performs some operations on the internal Nimbus database (which doesn't exist in the stateless version of nimbus)